docs-compose/overwatch_docs/content/DataEngineer/Definitions/071x.md

```diff
@@ -0,0 +1,1005 @@
+| warehouse_type            | string  | Warehouse type: PRO or CLASSIC                                                                                                                                                                                            |                                                                                                                                                                                           
+| warehouse_state           | string  | State of the warehouse                                                                                                                                                                                                    |                                                                                                                                                                                                   
+| size                      | string  | Size of the clusters allocated for this warehouse.                                                                                                                                                                        |                                                                                                                                                                       
+| creator_id                | long    | warehouse creator id                                                                                                                                                                                                      |                                                                                                                                                                                                     
+| tags                      | map     | A set of key-value pairs that will be tagged on all resources (e.g., AWS instances and EBS volumes) associated with this SQL warehouse.                                                                                   |                                                                                  
+| num_clusters              | long    | current number of clusters running for the service                                                                                                                                                                        |                                                                                                                                                                       
+| num_active_sessions       | long    | current number of active sessions for the warehouse                                                                                                                                                                       |                                                                                                                                                                      
```

Comment:
Might be useful to explain what a session is, or point to docs

Reply:
fetched the description from API docs